### PR TITLE
Perf tests for Skip default adapters

### PR DESCRIPTION
**Pull request by abhishkk**
## Description
Perf tests for skip default adapters.

### CODE REVIEW

File: test/Microsoft.TestPlatform.PerformanceTests/TranslationLayer/DiscoveryPerfTests.cs [hunk 1]
```diff
@@ -61,5 +61,47 @@ public void DiscoverNunit10K()
 
             this.PostTelemetry("DiscoverNunit10K", this.discoveryEventHandler2.Metrics);
         }
+
+        [TestMethod]
+        [TestCategory("TelemetryPerf")]
+        public void DiscoverMsTest10KWhenDefaultAdaptersSkipped()
```
Comment (smadala) [inline test/Microsoft.TestPlatform.PerformanceTests/TranslationLayer/DiscoveryPerfTests.cs]: Nit: DiscoverMsTest10KWithDefaultAdaptersSkipped?

File: test/Microsoft.TestPlatform.PerformanceTests/TranslationLayer/DiscoveryPerfTests.cs [hunk 2]
```diff
@@ -61,5 +61,47 @@ public void DiscoverNunit10K()
...
+        {
+            var testAssemblies = new List<string>
+                                     {
+                                         GetPerfAssetFullPath("MSTestAdapterPerfTestProject", "MSTestAdapterPerfTestProject.dll"),
+                                     };
+
+            this.vstestConsoleWrapper.DiscoverTests(testAssemblies, this.GetDefaultRunSettings(), new TestPlatformOptions() { CollectMetrics = true, SkipDefaultAdapters = true }, this.discoveryEventHandler2);
+
+            this.PostTelemetry("DiscoveryMsTest10K", this.discoveryEventHandler2.Metrics);
```
Comment (smadala) [inline test/Microsoft.TestPlatform.PerformanceTests/TranslationLayer/DiscoveryPerfTests.cs]: Change `DiscoveryMsTest10K` to `DiscoverMsTest10KWithDefaultAdaptersSkipped`.

Comment (abhishkk) [reply test/Microsoft.TestPlatform.PerformanceTests/TranslationLayer/DiscoveryPerfTests.cs]: done

File: test/Microsoft.TestPlatform.PerformanceTests/TranslationLayer/DiscoveryPerfTests.cs (hunk 1 above)
Comment (abhishkk) [reply test/Microsoft.TestPlatform.PerformanceTests/TranslationLayer/DiscoveryPerfTests.cs]: sone

**Review by smadala — APPROVED** [review]

**Pull request merged**